django_fido/management/commands/download_authenticator_metadata.py

```diff
@@ -137,7 +137,8 @@ def handle(self, **options):
                 self.stderr.write('Cannot determine the identificator from metadata response.')
                 continue
             url = authenticator_data['url']
-            authenticator, _ = AuthenticatorMetadata.objects.get_or_create(identifier=identifier)
+            authenticator, _ = AuthenticatorMetadata.objects.get_or_create(url=url)
```

Comment:
Shouldn't we change `AuthenticatorMetadata.url` to be unique, now that it's used as a primary identifier?

Also, what if there is more then one `url` for the same `identifier`? This would raise an exception, because `identifier` is supposed to be unique.

Reply:
Making the `url` unique is probably a good idea.

The mentioned situation probably cannot happen, as the identifier should be unique per authenticator make. So we might be OK with triggering an exception as that would signal a deeper issue with the MDS and its parsing.

Reply:
Also uniqueness of `identifier` is implied elsewhere, so dropping that constraint would cause issue there. I am for keeping that and having it raise an uncaught exception here.

Reply:
OK, sounds reasonable.